ui/apps/platform/src/Components/workflow/EntityTabs/EntityTabs.js

```diff
@@ -38,7 +39,11 @@ const EntityTabs = ({ entityType, activeTab }) => {
             relationshipTypes.CONTAINS,
             workflowState.useCase
         ),
-    ];
+    ]
+        // @TODO: Remove the following filter step once ROX_FRONTEND_VM_UPDATES is ON
+        .filter((match) => {
```

Comment:
nit: Wasn't sure if the word `match` made sense here. I think `containedEntity` makes more sense